packages/shared/src/components/InteractionCounter.tsx

```diff
@@ -24,7 +25,7 @@ export default function InteractionCounter({
   }, [value]);
 
   if (shownValue === value) {
-    return <>{shownValue}</>;
+    return <span {...props}>{shownValue}</span>;
```

Comment:
Just make sure this doesn't break any other implementations of this component.

Reply:
Yeah looking at tests now, usually span shouldn't break anything design wise, but 🤷‍♀️ ...